### PR TITLE
Add XML file for blocks, for WordPress 6.4

### PR DESCRIPTION
**Pull request by carolinan**
Partial for https://github.com/WordPress/theme-test-data/issues/83

I started with blocks generated on WordPress 6.4, so that they can be compared to the changes in 6.5.

The file includes a "how to" page that explains some more details:

### How to use the test data
The purpose of this test data is to test block changes across WordPress versions and Gutenberg versions. It is a compliment to the other [theme test data files](https://github.com/WordPress/theme-test-data).

A use case would be when you want to test the typography options of a block, and you don’t want to spend time adding the blocks and changing all the settings manually.

The blocks were generated using WordPress 6.4.3 with a block theme with the most permissive theme.json settings possible. All presets are the defaults: Colors, duotones, font sizes, spacing sizes, shadows, width etc.

Each block has its own post, and is listed under its category: Design, Theme, Widget etc.
Inner blocks such as the column, button, and comment blocks do not have their own posts.
Blocks that support borders have a tag called “Border”.

### Setting up
Download the file, activate the WordPress importer plugin and import the file.

- If you want to test the Author Biography block, you need to manually add a biography to the “theme reviewer” user.
- If you want to test the template only blocks, such as archive title, you need to add these manually to a template.

### Tips for testing
When you use block themes with different settings and templates, or a classic theme, you may find that not everything works as you expected. This is not necessarily a bug with the block, the theme, or the test file. If you suspect that there are bugs, it will require further testing.

For example:

- If your theme does not have a spacing preset called “7”, then spacing preset 7 will not work in your theme.
- If your theme does not support drop cap you will not be able to turn drop cap off and on.
- If the content block in your single template is not set to full width, then a full width block will not span the full width of the browser window.

### What’s not included
Experimental blocks
-   Forms
-   Time to read
-   Table of contents
-   Every embed block

Template only blocks
-   Archive and search title
-   Term description
-   Comment count
-   Comments link

### What it is not intended for

- Testing font families. No font families were included in this test, only default presets were used.
- Testing blocks on WordPress versions below 6.4.3
- Accessibility testing
- Testing private or scheduled posts, long titles, posts with many categories, etc.

### How you can help
You can help by reporting bugs, writing documentation, requesting features and submitting updates in the GitHub repository: https://github.com/WordPress/theme-test-data

### CODE REVIEW

**Comment by carolinan** [comment]
( Oops I forgot to create a separate branch, because I mistakingly tried to create the pull requests on Garys repository first. )

File: 64-block-test-data.xml [hunk 1]
```diff
@@ -0,0 +1,12170 @@
...
+</item>
+<item>
+	<title><![CDATA[How to use the test data]]></title>
+	<link>https://wpthemetestdata.wordpress.com/how-to/</link>
+	<pubDate>Wed, 07 Feb 2024 07:08:45 +0000</pubDate>
+	<dc:creator><![CDATA[themedemos]]></dc:creator>
+	<guid isPermaLink="false">https://wpthemetestdata.wordpress.com/how-to/</guid>
+	<description></description>
+	<content:encoded><![CDATA[<!-- wp:paragraph -->
+<p>The purpose of this test data is to test block changes across WordPress versions and Gutenberg versions. It is a compliment to the other <a href="https://github.com/WordPress/theme-test-data" data-type="link" data-id="https://github.com/WordPress/theme-test-data">theme test data files</a>.</p>
```
Comment (joyously) [inline 64-block-test-data.xml:2716]: compliment should be complement

**Review by kafleg — COMMENTED** [review]
Looks good to me. Import is successful without any error.

**Review by justintadlock — APPROVED** [review]
Overall, this looks pretty good. Imported with no issue. Lots of variety in testing content.

I did have an issue with the Author Biography post because the bio text wasn't appearing on the front end, but this is because I didn't have bio text in my test install (I assigned the imported posts to my admin account). Not sure how to address that other than make a note of it in the docs.

**Comment by carolinan** [comment]
Yeah the importer plugins doesn't use the author description.

File: 64-block-test-data.xml (hunk 1 above)
Comment (carolinan) [inline 64-block-test-data.xml:2716]:
```suggestion
<p>The purpose of this test data is to test block changes across WordPress versions and Gutenberg versions. It is a complement to the other <a href="https://github.com/WordPress/theme-test-data" data-type="link" data-id="https://github.com/WordPress/theme-test-data">theme test data files</a>.</p>
```

Comment (carolinan) [reply 64-block-test-data.xml]: Thank you.

**Pull request merged**